src/lib.rs

```diff
@@ -12,8 +12,6 @@
 #![allow(clippy::type_complexity)]
 #![cfg_attr(docsrs, feature(doc_cfg))]
 #![cfg_attr(feature = "simd", feature(portable_simd))]
-#![feature(build_hasher_simple_hash_one)]
```

Comment:
This has performance benefits.

We could add this behind a nightly feature (and maybe activate that feature in a build.rs file if a nightly compiler is detected).

We can write a custom trait to deal with the dispatch:

```rust
use std::hash::{BuildHasher, Hash};

pub trait HashSingle: BuildHasher {
    #[inline]
    fn hash_single<T: Hash>(&self, x: T) -> u64
    where
        Self: Sized,
    {
        #[cfg(feature = "nightly")]
        {
            self.hash_one(x)
        }
        #[cfg(not(feature = "nightly"))]
        {
            use std::hash::Hasher;
            let mut hasher = self.build_hasher();
            x.hash(&mut hasher);
            hasher.finish()
        }
    }
}

impl<T: BuildHasher> HashSingle for T {}

```